apps/api/src/app/shared/helpers/base62.ts

```diff
@@ -0,0 +1,22 @@
+import basex from 'base-x';
+
+const BASE62 = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+const { encode, decode } = basex(BASE62);
+
+export function encodeBase62(value: string): string {
+  const buffer = Buffer.from(value, 'hex');
+
+  return encode(buffer);
+}
+
+export function decodeBase62(encoded: string): string {
+  const uint8Array = decode(encoded);
+
+  return Buffer.from(uint8Array).toString('hex');
+}
+
+export function isBase62(input: string): boolean {
+  const base62Regex = /^[0-9A-Za-z]+$/;
+
+  return base62Regex.test(input);
+}
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** Beware, [`base-x` has some logic](https://github.com/cryptocoinjs/base-x/blob/64e196c84f1d0b51eb7650d3881868568a0f525c/ts_src/index.ts#L39) to "skip" leading zeros when encoding/decoding. We should add tests for this case.‏

Reply:
mongo object is the first byte of the timestamp component, and the timestamp component is always a positive integer. however in order to make sure i added tests, from the tests i found out that the encoding supports 2 leading zero.